Jenkinsfile_nightly

```diff
@@ -46,6 +46,7 @@ withNightlyPipeline(type, product, component) {
 
   before('fullFunctionalTest') {
     env.PROXY = params.PROXY_SERVER
+    env.PARALLEL_CHUNKS = 10
```

Comment:
Want to add it as a build param?

Reply:
can be added, but not sure if needed

Reply:
done, but looks that numeric types are not supported

Reply:
Wasn't sure either but just mentioning the possibility

Reply:
> done, but looks that numeric types are not supported

Yeah seems that way from [here](https://www.jenkins.io/doc/book/pipeline/syntax/#parameters). Do you think that any conversions will need to take place or javascript will handle that all for us (thinking the latter but want to double check)